Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -271,7 +273,11 @@ - (void)startScanningWithResultBlock:(void (^)(NSArray *codes))resultBlock error
         if (self.didStartScanningBlock) {
             self.didStartScanningBlock();
         }
+
+        return YES;
     }
+
+    return NO;
```

Comment:
Wasn't sure if it would be merged upstream so I wanted to keep the diff minimal.

Will change when doing the dispatch refactoring.